lib/get_opts.py

```diff
@@ -38,6 +38,7 @@
 reset_parser.add_argument('scope', help='things to reset: all, scripts, data, or connections', choices=['all', 'scripts', 'data', 'connections'])
 install_parser.add_argument('--compile', help='force re-compile of script before downloading', action='store_true')
 install_parser.add_argument('--debug', help='run in debug mode', action='store_true')
+install_parser.add_argument('--not-cached', help='clears local cache before installation', action='store_true')
```

Comment:
Technically this implementation overwrites the local cache, so maybe something like `overwrites local cache of raw data`